package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "proto3-json-serializer",
+  "version": "0.1.0",
+  "repository": "googleapis/proto3-json-serializer-nodejs",
+  "description": "Support for proto3 JSON serialiazation/deserialization for protobuf.js",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [
+    "protobufjs",
+    "protobuf.js",
+    "protobuf",
+    "proto3",
+    "json",
+    "serialization",
+    "deserialization"
+  ],
+  "scripts": {
+    "test": "c8 node -r source-map-support/register node_modules/mocha/bin/mocha build/test/unit",
```

Comment:
Source map support is built in to newer versions of Node.js now, so shouldn't need `source-map-register`:

```
--enable-source-maps
```

Reply:
Thanks! Some random stackoverflow article I found this in must be updated :)